README.adoc

```diff
@@ -33,8 +33,8 @@ strategies such as canary deployments provide the safety necessary to capture va
  
 = Tech Specs
 
-Spinnaker consists of a number of https://www.spinnaker.io/reference/architecture/[independent microservices], with the https://github.com/spinnaker/halyard[Halyard CLI tool] or the https://docs.armory.io/spinnaker/operator/[Kubernetes Operator] (Beta)
-managing the lifecycle of the other services. A https://www.spinnaker.io/setup/other_config/[variety of SDLC tools] integrate with Spinnaker, and its plugin framework makes Spinnaker more easily customizable to your needs. Plugins allow us to add system integrations without updating Spinnaker, broadening its potential
+Spinnaker consists of a number of https://www.spinnaker.io/reference/architecture/[independent microservices], with the https://github.com/spinnaker/halyard[Halyard CLI tool] or the https://docs.armory.io/docs/[Kubernetes Operator] (Beta)
```

Comment:
Isn't https://www.armory.io/spinnaker-community/spinnaker-operator/ a more correct link for Armory's kubernetes operator?

Reply:
Yes I cross checked this too but lent towards top level link.
Guess deep link or prefix with `Armory's` perhaps to indicate why redirecting to a different organization.

Reply:
Mentioning Armory specifically makes sense to me.

Reply:
How about I add a link to both actually? One for top-level and one for spinnaker specific? I couldn't find the spinnaker specific input while I was looking previously so that would be a valuable direct link to have.

Reply:
I've amended the content by updating to @dbyron-sf 's link. I may have messed up on the commit message naming convention though. If squash and merge isn't an option I'll have to go do the rebase and force-push stuff on my end ... probably tomorrow night.

Deep linking to the spinnaker specific armoury operator makes sense, since anyone not familiar with armory could click UP in the documentation breadcrumbs, while anyone already familiar with armory would just want to go direct to the spinnaker-specific part of the docs.

Reply:
LGTM, thanks!